components/feature/addons/src/test/java/mozilla/components/feature/addons/worker/ExtensionsTest.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.addons.worker
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import junit.framework.TestCase.assertTrue
+import java.io.IOException
+import kotlinx.coroutines.CancellationException
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Indicates if an exception should be reported to the crash service.
+ */
+internal fun Exception.shouldReport(): Boolean = cause is IOException || this is CancellationException
```

Comment:
I assume you meant to import this function? Otherwise you're testing the test code :).

Reply:
🤦‍♂  it is there by mistake

Reply:
I updated the pr 👍